src/avatar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cn from 'classnames';
+import img from './missing.png';
+
+import styles from './styles.css';
+
+export default ({ src = '', inline = '', defaultImage = img }) => {
+  const image = src || defaultImage;
+  const classes = cn(
+    styles['avatar-image'],
+    inline ? styles.inline : '',
```

Comment:
maybe? I think ternary reads better if `inline` is the only variable and its bool